### PR TITLE
Update metadata.yaml for Nafisa Jadavji

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Changes made by author, I'm just opening the PR

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -363,7 +363,7 @@ authors:
       - Writing - Original Draft
       - Writing - Review & Editing
     code of conduct:
-      confirmed: No
+      confirmed: Yes
     affiliations:
     - Biomedical Science, Midwestern University, Glendale, AZ, United States of America
     - Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
```
Comment (rando2) [inline content/metadata.yaml:369]: @nafisajadavji would you be able to change the date below to 2020-11-11? Or else if you confirm your conflict of interest is still correct, I can make the change separately!

Comment (nafisajadavji) [reply content/metadata.yaml:369]: @rando2 Sorry for the confusion, I am still figuring out Github. I think I did what you asked me to.

Comment (rando2) [reply content/metadata.yaml:369]: You did!! Thank you so much! I know it's super tricky, especially with the .yaml file which isn't a totally common format off of GitHub!

**Comment by rando2** [comment]
Once again I'm not quite sure how to link this up with the commit, but 49415c7 adds these changes.

**Pull request closed without merging**